### PR TITLE
explicitly show plugins as unsupported on !linux

### PR DESCRIPTION
**Pull request by vieux**
Fix https://github.com/docker/docker/issues/28281

ping @anusha-ragunathan @tiborvass @jhowardmsft

### CODE REVIEW

**Comment by tiborvass** [comment]
design +1 and code LGTM

**Comment by jhowardmsft** [comment]
Seems reasonable. Don't have a way of testing it though...

**Review by vdemeester — APPROVED** [review]
LGTM :frog:

File: plugin/backend_linux.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// +build linux
```
Comment (cpuguy83) [inline plugin/backend_linux.go]: No need for this.

**Comment by anusha-ragunathan** [comment]
LGTM

**Comment by mavenugo** [comment]
@anusha-ragunathan @vieux is it true for plugin-v1 as well ?

@jhowardmsft can you pls confirm if you know if any windows plugin for the v1 format (which was very liberal in its approach) ?

**Comment by anusha-ragunathan** [comment]
@mavenugo : pluginv1 will be supported on Windows and Linux. This PR is specifically for pluginv2 on Windows.

**Comment by jhowardmsft** [comment]
@mavenugo I don't know. @patricklang? ^^

**Comment by mavenugo** [comment]
Thanks @anusha-ragunathan. That makes it clear :) Maybe we should document that clearly to avoid confusions.

**Comment by anusha-ragunathan** [comment]
@vieux : It might be worth it to add a statement to https://github.com/docker/docker/blob/master/docs/extend/index.md regarding supported daemon OS.

@mavenugo : Regarding existing pluginv1s on Windows, @anaghab12 from Nimble should be able to confirm the effort they had on supporting Windows plugins. Refer https://github.com/docker/docker/issues/23605 for details.

**Comment by thaJeztah** [comment]
Opened a PR for adding a note; https://github.com/docker/docker/pull/28715

**Comment by anusha-ragunathan** [comment]
@thaJeztah : Thanks for doing this.

**Pull request merged**